packages/base-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.0.3]
+
+### Uncategorized
```

Comment:
Looks like the change entries haven't been categorized yet. Each change entry should be grouped into a category. See step 2 here: https://github.com/MetaMask/core/blob/main/docs/contributing.md#releasing

Reply:
Thank you. Done.